### PR TITLE
make etcd-starter -version special

### PR DESCRIPTION
**Pull request by xiang90**
/cc @crawford @yichengq @kelseyhightower

### CODE REVIEW

**Comment by xiang90** [comment]
```
ETCD_INTERNAL_BINARY_DIR=./ ./etcd-starter -version
etcd-starter version 0.0.2+git
internal version 1: etcd version 0.4.7+git
internal version 2: etcd version 2.0.4+git
```

```
ETCD_INTERNAL_BINARY_DIR=./ ./etcd -version
2015/03/24 18:04:45 starter: data-dir is not set.
2015/03/24 18:04:45 starter: starting etcd version 2
2015/03/24 18:04:45 starter: starting with 2/etcd [-version] with env [ETCD_INTERNAL_BINARY_DIR=./ TERM_PROGRAM=Apple_Terminal SHELL=/bin/bash TERM=xterm-256color TMPDIR=/var/folders/jr/0f3rv12516s2z2xh2m_4fn540000gn/T/ Apple_PubSub_Socket_Render=/private/tmp/com.apple.launchd.vNNu29YBrg/Render TERM_PROGRAM_VERSION=343.6 OLDPWD=/Users/xiangli/go/src/github.com/coreos TERM_SESSION_ID=1AE6F463-FEDC-416F-ABE9-9C160E7EEE37 USER=xiangli SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.M63oZ6ELQ4/Listeners __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x0 PATH=/Users/xiangli/google-cloud-sdk/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/Users/xiangli/go/bin PWD=/Users/xiangli/go/src/github.com/coreos/etcd-starter LANG=en_US.UTF-8 XPC_FLAGS=0x0 XPC_SERVICE_NAME=0 SHLVL=1 HOME=/Users/xiangli LOGNAME=xiangli GOPATH=/Users/xiangli/go SECURITYSESSIONID=186a5 _=./etcd]
etcd version 2.0.4+git
```

File: starter/starter.go [hunk 1]
```diff
@@ -108,7 +103,8 @@ func checkInternalVersion(fs *flag.FlagSet) version {
 
 	dataDir := fs.Lookup("data-dir").Value.String()
 	if dataDir == "" {
-		log.Fatalf("starter: please set --data-dir or ETCD_DATA_DIR for etcd")
+		log.Printf("starter: data-dir is not set")
```
Comment (philips) [inline starter/starter.go]: etcd-starter for consistency

File: starter/starter.go [hunk 2]
```diff
@@ -66,10 +65,6 @@ func StartDesiredVersion(binDir string, args []string) {
 	if err != nil {
 		return
 	}
-	if fs.Lookup("version").Value.String() == "true" {
-		fmt.Println("etcd version", etcdversion.Version)
-		os.Exit(0)
-	}
 
 	ver := checkInternalVersion(fs)
 	log.Printf("starter: starting etcd version %s", ver)
```
Comment (philips) [inline starter/starter.go]: etcd-starter for consistency

**Comment by crawford** [comment]
LGTM

**Comment by philips** [comment]
Only major nit is getting rid of the log timestamps.

File: starter/starter.go (hunk 1 above)
Comment (kelseyhightower) [inline starter/starter.go]: What if the default data dir chosen by etcd actually already exits with 0.4.x contents?

**Comment by xiang90** [comment]
@philips All resolved. PTAL.

**Comment by philips** [comment]
+1

**Comment by kelseyhightower** [comment]
LGTM

**Pull request merged**